x-pack/plugins/asset_manager/server/plugin.ts

```diff
@@ -15,26 +15,48 @@ import {
   PluginConfigDescriptor,
   Logger,
 } from '@kbn/core/server';
+
 import { upsertTemplate } from './lib/manage_index_templates';
+import { startImplicitCollection } from './lib/implicit_collection';
 import { setupRoutes } from './routes';
 import { assetsIndexTemplateConfig } from './templates/assets_template';
 
 export type AssetManagerServerPluginSetup = ReturnType<AssetManagerServerPlugin['setup']>;
-export interface AssetManagerConfig {
-  alphaEnabled?: boolean;
-}
+
+const configSchema = schema.object({
+  alphaEnabled: schema.maybe(schema.boolean()),
+  implicitCollection: schema.maybe(
+    schema.object({
+      enabled: schema.maybe(schema.boolean({ defaultValue: true })),
+      interval: schema.duration({ defaultValue: '5m' }),
```

Comment:
enabled/disabled and collection interval might be good candidates to move to Advanced Settings eventually (not now)